app/config/__init__.py

```diff
@@ -246,6 +260,19 @@ class LocalConfig(_SharedConfig):
     PROXY_FIX_HOST: int = 0
     SERVER_NAME: str = "funding.communities.gov.localhost:8080"
 
+    # The LocalConfig needs default values for these AZURE_AD variables so that the Check DB Migrations job
+    # can run correctly in CI but these should be overwritten in your local .env file with real values
+    # if you want to sign in with SSO locally
+
+    # Azure Active Directory Config
+    AZURE_AD_CLIENT_ID: str = "00000000-0000-0000-0000-000000000000"
+    AZURE_AD_CLIENT_SECRET: str = "incorrect_value"
+    AZURE_AD_TENANT_ID: str = "00000000-0000-0000-0000-000000000000"
+
+    @property
+    def AZURE_AD_AUTHORITY(self) -> str:
```

Comment:
Can this be overridden by setting an env var? Can we document how devs can use real Microsoft sso locally?